Cairo Desktop/Cairo Desktop/TaskButton.xaml

```diff
@@ -83,7 +83,7 @@
                 </MenuItem>
             </ContextMenu>
         </Button.ContextMenu>
-        <Grid>
+        <Grid x:Name="ButtonGrid">
```

Comment:
I believe this change should no longer be necessary.

Reply:
All set.